book/index.rst

```diff
@@ -16,6 +16,7 @@ Be sure to know a thing or two about `Symfony`_. Sulu is based on Symfony and st
     twig
     content-architecture
     webspaces
+    snippets
```

Comment:
I think I would not add an additional chapter in the book, because a lot of projects dont need to use snippets. Instead i would structure it like this:

1. Add a section below the `Content Blocks` section in the templates chapter: https://docs.sulu.io/en/2.2/book/templates.html#content-blocks
2. Shortly explain snippets and its usecases in the section and add a link to the`SnippetBundle` documentation
3. Create the `SnippetBundle` documentation (similar to the `LocationBundle`) https://docs.sulu.io/en/latest/bundles/location.html 

What do you think about this? 🙂

Reply:
See your point. 

I generally wanted to expand on the snippets a bit more. 
However, I have decided for time reasons to create a first basic version as PR and add the rest after. 

Therefore the decision of a separate chapter.

but im interestet in your opinion, because of me, we can also gladly transfer the content to another chapter. :) !

Reply:
yeah, i think it would make sense to create a `SnippetBundle` documentation similar to the `LocationBundle` for this 🙂